chorus_lib/src/core.rs

```diff
@@ -37,12 +37,11 @@ impl Superposition for () {
 ///
 /// The struct takes two type parameters: `V` and `L1`.
 ///
-/// - `V` is an actual type of the value. It must satisfy the `ChoreographicValue` trait.
+/// - `V` is an actual type of the value.
 /// - `L1` is the location of the value. It must satisfy the `ChoreographicLocation` trait.
 #[derive(PartialEq)]
 pub struct Located<V, L1>
 where
-    V: ChoreographicValue,
```

Comment:
`Located` can hold non-portable values. For example, we can imagine a choreography that takes a connection to a database (which obviously cannot be transmitted to another location) and uses it inside the choreography. Therefore, the `Portable` bound should only be imposed on `comm` and `broadcast`.